acl_loader/main.py

```diff
@@ -173,7 +173,31 @@ def read_tables_info(self):
         Read ACL_TABLE table from configuration database
         :return:
         """
-        self.tables_db_info = self.configdb.get_table(self.ACL_TABLE)
+        # get the acl table info from host config_db
+        host_acl_table = self.configdb.get_table(self.ACL_TABLE)
+        # For multi asic get only the control plane acls from the host config_db
+        if self.per_npu_configdb:
+            for table, entry in host_acl_table.items():
+                if entry.get('type', None) != self.ACL_TABLE_TYPE_CTRLPLANE:
```

Comment:
good to add .upper() while comparing with  self.ACL_TABLE_TYPE_CTRLPLANE
The same comment at line num 194 below